src/main/java/com/thevoxelbox/voxelsniper/brush/Brush.java

```diff
@@ -200,6 +194,18 @@ protected final World getWorld() {
         return targetBlock.getWorld();
     }
 
+    protected final int getMinHeight() {
```

Comment:
Should probably replace all `getWorld().getMinHeight()` calls with `getMinHeight()` - but we can do that later, or over time.
(same for getMaxHeight)

Reply:
Didn't I already do that?

Reply:
I think I still found a few, will do it after this PR. InteliJ finds 6 of only the max, min is probably similar.

Reply:
did some more in latest commit